op-deployer/pkg/deployer/pipeline/start_block.go

```diff
@@ -24,12 +24,17 @@ func SetStartBlockLiveStrategy(ctx context.Context, env *Env, st *state.State, c
 	if err != nil {
 		return fmt.Errorf("failed to get start block: %w", err)
 	}
-	thisChainState.StartBlock = startHeader
+	thisChainState.StartBlock = &genesis.BlockRef{
```

Comment:
use the RPC client from state to populate the blockref directly:

`rpcClient.CallCtx(ctx, "eth_getHeaderByNumber", "latest", &blockRef)`